src/components/FileTree.tsx

```diff
@@ -626,6 +634,10 @@ export const FileTreeInner = ({
     FileEntry | undefined
   >(undefined)
 
+  const [treeSelection, setTreeSelection] = useState<FileEntry | undefined>(
+    loaderData.file
+  )
+
```

Comment:
I'm dumb, I was going to do this, but I didn't, in fear I had to traverse the tree to deselect other shit :facepalm: